component/app.js

```diff
@@ -53,5 +63,70 @@ class App {
 
     start () {
         this.getGrades();
+        this.gradeForm.onSubmit(this.createGrade)
+        this.gradeTable.onDeleteClick(this.deleteGrade)
+
+    }
+
+    
+    createGrade (name, course, grade) {
+        console.log("in createGrade", "name: ", name, "course: ", course, "grade: ", grade);
+         
+        var ajaxOptions = {
+             url: "https://sgt.lfzprototypes.com/api/grades",
+             method: "POST",
+             data:
+             { 
+                 "name": name,
+                 "course": course,
+                 "grade" : grade
+             },            
+             dataType: "json",        
+             headers: {
+                 "X-Access-Token": "944Roegi"
+             },
+             success: this.handleCreateGradeSuccess,
+             error: this.handleCreateGradeError
+         };
+     
+             return $.ajax(ajaxOptions);
+     }
+
+    handleCreateGradeSuccess(){
+        this.getGrades();
+    }
+
+    handleCreateGradeError() {
+        console.error(error);
+    }
+
+
+
+    deleteGrade(id) {
+        console.log("id: ", id);

```

Comment:
In general, you'll want to remove testing `console.log`s after you're done verifying their functionality.